packages/wrangler/src/config/index.ts

```diff
@@ -264,16 +264,21 @@ export function printBindings(
 				({ name, class_name, script_name }) => {
 					let value = class_name;
 					if (script_name) {
-						const registryDefinition = context.registry?.[script_name];
-						if (
-							registryDefinition &&
-							registryDefinition.durableObjects.some(
-								(d) => d.className === class_name
-							)
-						) {
-							value += ` (defined in 🟢 ${script_name})`;
+						if (context.local) {
+							const registryDefinition = context.registry?.[script_name];
+
+							if (
+								registryDefinition &&
+								registryDefinition.durableObjects.some(
+									(d) => d.className === class_name
+								)
+							) {
+								value += ` (defined in 🟢 ${script_name})`;
+							} else {
+								value += ` (defined in 🔴 ${script_name})`;
```

Comment:
now that I've read the actual issue 😅 , I'm seeing that our users' confusion is around using the red bullets. Do we still want to use them in this place in that case?

Reply:
I think this is a good first step—it removes them from the dploy flow, which is where I think most confusion was coming from

Reply:
kk